src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntTests.cs

```diff
@@ -39,6 +41,100 @@ public void LanguageVersion()
             comp.VerifyDiagnostics();
         }
 
+        [Fact]
+        public void FromSource()
```

Comment:
>FromSource [](start = 20, length = 10)

It doesn't look like tests for PE are added. It Also doesn't look like we explicitly test Equals/GetHashCode